Core/MIPS/ARM64/Arm64Asm.cpp

```diff
@@ -113,9 +115,109 @@ void Arm64Jit::GenerateFixedCode(const JitOptions &jo) {
 		loadStaticRegisters = nullptr;
 	}
 
-	enterCode = AlignCode16();
+	applyRoundingMode = AlignCode16();
```

Comment:
I guess it doesn't matter, but why not move this down like the integer rounding funcs?  Then you don't even need to monkey with start any more than already.

-[Unknown]